src/hw13/main.java

```diff
@@ -0,0 +1,16 @@
+package hw13;
+
+public class main {
+    public static void main(String[] args) {
+        Printable[] mass = new Printable[]{new Book(),new Magazine()};
+        for(Printable x:mass){
+             x.print();
+        }
+
```

Comment:
use method  printMagazines(Printable[] printable)  and printBooks(Printable[] printable) at least once